Makefile

```diff
@@ -20,7 +20,7 @@ GATEKEEPER_VERSION = v3.15.1
 GOTESTSUM = gotest.tools/gotestsum@v1.11.0
 
 # Golang version go mod tidy compatibility
-GOLANG_VERSION ?= 1.21
```

Comment:
If we can, let's remove this? Recent Go versions are explicit about the version and toolchain in `go.mod` itself.

Reply:
addressed